.github/workflows/coding-conventions.yml

```diff
@@ -35,8 +35,8 @@ jobs:
       - name: Run eslint
         run: bun run lint
 
-  phpstan:
-    name: "Run Static Analysis: PHPStan"
+ phpstan:
+    name: "Run static analysis: phpstan"
```

Comment:
It looks to me like you are adopting the brand casing for GitHub, Bun, etc. in this PR. We should probably do the same for __PHPStan__.

Reply:
Absolutely. Good catch!